### DISCUSSION TITLE
RpcIpcMessagePortClosedError

### DISCUSSION THREAD

**Discussion by bhutanict** [apache/superset#18334] [Q&A / Help]
#### How to reproduce the bug

Clone the lastest superset
run npm install in the superset-frontend
run npm run dev-server

### Expected results

Run the superset-frontend project successfully 

### Actual results

Error

#### Screenshots

<img width="1119" alt="Screen Shot 2021-10-20 at 3 27 30 PM" src="https://user-images.githubusercontent.com/12108741/138068139-c1f14f2e-16df-41b4-ad80-05ed3b01c7c6.png">

### Environment

(please complete the following information):

- browser type and version:
- superset version: `lastest cloned on 20/10/2021`
- python version: `python 3.8`
- node.js version: `16.9.1`
- npm: '7.21.1'

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [Y] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [Y] I have reproduced the issue with at least the latest released version of superset.
- [Y] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by piyushdatazip** [comment]
Hi I am facing the same issue, any help?

**Comment by VillaltaJose** [comment]
Hi I am facing the same issue when running the build commando, any help?

**Comment by tshallenberger** [comment]
**[SOLUTION]** 
I encountered an identical issue running `npm run build` for Superset v3.1.3 in a VS Code DevContainer. 
Made the following change in the `.devcontainer/docker-compose.yml`:

```yaml
services:
  app:
    # mem_limit: 8gb
    mem_limit: 12gb 
```

You'll notice that in `superset-frontend/webpack.config.js`, this line [here](https://github.com/apache/superset/blob/b477abd1ca8dde4ef6d8d8a19c6bdb4d51cbe8eb/superset-frontend/webpack.config.js#L160) sets up the `ForkTsCheckerWebpackPlugin` with a `memoryLimit` of `4096`. I'm gonna assume if you run into this issue you don't have enough memory allocated for `npm run build`.
